R/utilities.r

```diff
@@ -90,7 +90,7 @@ remove_missing <- function(df, na.rm = FALSE, vars = names(df), name = "",
   missing <- detect_missing(df, vars, finite)
 
   if (any(missing)) {
-    df <- df[!missing, ]
+    df <- df[!missing, , drop = FALSE]
```

Comment:
This shouldn't occur in wild plotting because there should always be multiple columns. However, since it came up in testing, I thought to change this too.

Reply:
Agreed - I think we should aim to make the code as clear and type stable as possible